internal/pkg/store/secretsmanager/secretsmanager.go

```diff
@@ -61,6 +61,15 @@ func (s *SecretsManager) CreateSecret(secretName, secretString string) (string,
 	return aws.StringValue(resp.ARN), nil
 }
 
+func (s *SecretsManager) DeleteSecret(secretName string) error {
+	_, err := s.secretsManager.DeleteSecret(&secretsmanager.DeleteSecretInput{
```

Comment:
I think we should probably deal with idempotency here, since `DeleteSecret` throws `ResourceNotFoundException` when secret does not exist. Ideally it would be better to keep going if a resource has been deleted instead of aborting.

Reply:
Should be resolved by 3b9bf4c